### ISSUE TITLE
About Save Files(.pt) and Reload Inference Accuracy

### ISSUE THREAD

**Issue by xiaoLiuxiaoLiuxiaoLiu** [pytorch/pytorch#28572]
## 🐛 Bug

There is a problem happened on my case at Libtorch1.1.0
Training and Directly Inference get a normal and correct result, however, Training-Save and Load-Inference get a innormal and competely wrong result.
Any HELP?

Here is my flow path of my code, Step 6 outputs are same but Step 7 not and competely different:

## To Reproduce

1. Model Initialization
2. Model Training(printf:output result with an all-ones intput at last training step)
3. torch::save(model,“D:\path\to\model.pt”);
4. Initialization model2
5. torch::load(model2,“D:\path\to\model.pt”);
6. std::cout<parameters()[i]<<std::endl;
    std::cout<parameters()[i]<<std::endl;
7. torch::Tensor t1=model->forward(torch::ones(1,1,224,224).to(torch::kCUDA));
    torch::Tensor t2=model2->forward(torch::ones(1,1,224,224).to(torch::kCUDA));

## Expected behavior
Step 7 outputs should be same.

## Environment

 - Libtorch1.1.0 CUDA-Release and Debug:
 - Windows10:
 - C++ with VS2017:
 - CUDA/cuDNN 10.0/7.6:
 - GPU models: ShuffleNetV2_x0_5 and ShuffleNetV1_x1_0

cc @ezyang @gchanan @zou3519 @jerryzh168

**Comment by zhangguanheng66** [comment]
Mark as high priority because of the inconsistent behaviors of `torch.load()` and `torch.save()`.

**Comment by zhangguanheng66** [comment]
@xiaoLiuxiaoLiuxiaoLiu Please provide us a script to reproduce the bug.

**Comment by gchanan** [comment]
actually, I'm just going to kill high priority, there's no information to go off of here.

**Comment by zhangguanheng66** [comment]
If we can reproduce the error with @xiaoLiuxiaoLiuxiaoLiu 's help, it could be a high priority issue.

**Comment by xiaoLiuxiaoLiuxiaoLiu** [comment]
@gchanan Hello, last night I supposed this bug killed. It is a bug about unreasonable Program Design due to without ptr to eval() and train() of the model on my Reloading Class. So there exists quite different output result. I suggested that this issue can be closed. 
Thank You！

**Comment by xiaoLiuxiaoLiuxiaoLiu** [comment]
@zhangguanheng66 Hello, last night I supposed this bug killed. It is a bug about unreasonable Program Design due to without ptr to eval() and train() of the model on my Reloading Class. So there exists quite different output result. I suggest that this issue can be closed. 
Thank You！

**Comment by ezyang** [comment]
OK, thanks for the update~
